### PR TITLE
Created Anna page and content

### PR DESCRIPTION
**Pull request by JohnMeyer32**
This is Anna Akhmatova's bio page. This is the last author i added for the initial bio pages and should be good to merge other than a link to the home page. This is part of the issues 7 & 8:
https://github.com/lukeware/WATS-4020-Team-2/issues/7
https://github.com/lukeware/WATS-4020-Team-2/issues/8
Part 4/4

### CODE REVIEW

**Comment by Hazel61** [comment]
Looks good. No conflicts with master. Merge ahead.

**Review by Hazel61 — APPROVED** [review]
Hi John, looks great. No conflicts, so merge already.

**Review by dougjones866 — APPROVED** [review]
Everything looks all good, merge on.

**Comment by Okideveloper** [comment]
looks good!

File: Anna-Akhmatova-Bio.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    </div>
+
+    <p>
+        <button class="btn btn-primary" type="button" data-toggle="collapse" data-target="#EAP-list-of-works" aria-expanded="false" aria-controls="EAP-list-of-works">
+            List of Works
+        </button>
+    </p>
+    <div class="collapse" id="EAP-list-of-works">
+        <div class="card card-body">
+            <ul>
```
Comment (Hazel61) [inline Anna-Akhmatova-Bio.html:114]: Would be great to have links to these poems

**Pull request closed without merging**